packages/features/ee/workflows/api/scheduleEmailReminders.ts

```diff
@@ -119,6 +119,19 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
         switch (reminder.workflowStep.action) {
           case WorkflowActions.EMAIL_HOST:
             sendTo = reminder.booking?.userPrimaryEmail ?? reminder.booking.user?.email;
+            const hosts = reminder.booking.attendees
+              .filter((attendee) =>
+                reminder.booking?.eventType?.hosts?.some((host) => host.user.email === attendee.email)
+              )
+              .map((host) => host.email);
+            const schedulingType = reminder.booking.eventType?.schedulingType;
+
+            if (
+              hosts &&
+              (schedulingType === SchedulingType.COLLECTIVE || schedulingType === SchedulingType.ROUND_ROBIN)
+            ) {
+              sendTo = sendTo ? [sendTo, ...hosts] : hosts;
+            }
```

Comment:
This looks fine but what if user has connected a calendar. in that case we send the email and create calendar invite only on the connected calendar email address

Reply:
Not sure wdym but that. Since user is host wouldn't he already have calendar invite? Also isn't calendar.ics being sent with the mail?

Reply:
email address in user' account could be different than connected calendar email